ddtrace/internal/utils.py

```diff
@@ -0,0 +1,28 @@
+from ddtrace.compat import parse
+from ddtrace.vendor.dogstatsd import DogStatsd
+
+
+def _parse_dogstatsd_url(url):
```

Comment:
I wonder if this should go in utils/formats 🤔

Reply:
I kept this together with the dogstatsd connection handling as it doesn't seem generally applicable. I could see a benefit from refactoring the url parsing for the trace and metrics.